go/libkb/env.go

```diff
@@ -455,6 +457,20 @@ func (e *Env) GetSocketFile() (ret string, err error) {
 	return
 }
 
+func (e *Env) GetGregorURI() string {
+	return e.GetString(
+		func() string { return os.Getenv("GREGOR_URI") },
+		func() string { return e.config.GetGregorURI() },
+	)
+}
+
+func (e *Env) GetGregorUseTLS() bool {
+	return e.GetBool(false,
+		func() (bool, bool) { return e.getEnvBool("GREGOR_USE_TLS") },
```

Comment:
if we have a URI scheme above, we might not need a separate config parameter.  if we do have a config parameter for this alone, we should have a sensible default